packages/calcite-components/src/components/popover/popover.stories.ts

```diff
@@ -186,25 +186,3 @@ export const largeScaleLayout_TestOnly = (): string => html`
     </calcite-popover>
   </div>
 `;
-
-export const transparentBG_TestOnly = (): string => html`
```

Comment:
We'll need to restore this one since it added coverage for https://github.com/Esri/calcite-design-system/issues/6798.

Reply:
@alisonailea Are we ok restoring this?

Reply:
It's covered by theming tests isn't it?